lib/erl/src/thrift_compact_protocol.erl

```diff
@@ -355,7 +356,7 @@ read(This0, i64) ->
 read(This0, double) ->
   {This1, Bytes} = read_data(This0, 8),
   case Bytes of
-    {ok, <<Val:64/float-signed-big, _/binary>>} -> {This1, {ok, Val}};
+    {ok, <<Val:64/float-signed-little, _/binary>>} -> {This1, {ok, Val}};
```

Comment:
Why is "float" used here instead of "Double"?  I don't know erlang; just looking at line 222 above and expecting it to be similar.